### ISSUE TITLE
typing `help(config)` in the demo mutates it

### ISSUE THREAD

**Issue by dvd101x** [josdejong/mathjs#3141]
Hi, found this curious behavior on both the demo and mathnotepad.

Typing `help(config)` shows the help but also runs the examples. The examples seem to be running on the same math instance as the demo. So the demo is reconfigured by running help.

```jl
help(config)

# Name: config
# Category: Core
# 
# 
# Description:
#     Get configuration or change configuration.
# 
# Syntax:
#     config()
#     config(options)
# 
# Examples:
#     config()
#         {"epsilon": 1e-12, "matrix": "Matrix", "number": "Fraction", "precision": 64, "predictable": false, "randomSeed": null}
#     1/3 + 1/4
#         7/12
#     config({number: "Fraction"})
#         {"epsilon": 1e-12, "matrix": "Matrix", "number": "Fraction", "precision": 64, "predictable": false, "randomSeed": null}
#     1/3 + 1/4
#         7/12

1
# 1/1
```

**Comment by josdejong** [comment]
Fixed now in `v12.3.1`, thanks for reporting!

**Comment by dvd101x** [comment]
thank you Jos!
